egs/libritts/tts1/run.sh

```diff
@@ -0,0 +1,250 @@
+#!/bin/bash
+
+# Copyright 2019 Nagoya University (Takenori Yoshimura)
+#  Apache 2.0  (http://www.apache.org/licenses/LICENSE-2.0)
+
+. ./path.sh
+. ./cmd.sh
+
+# general configuration
+backend=pytorch
+stage=-1
+stop_stage=100
+ngpu=1       # number of gpu in training
+nj=32        # numebr of parallel jobs
+dumpdir=dump # directory to dump full features
+verbose=0    # verbose option (if set > 1, get more log)
+seed=1       # random seed number
+resume=""    # the snapshot path to resume (if set empty, no effect)
+
+# feature extraction related
+fs=24000      # sampling frequency
+fmax=""       # maximum frequency
+fmin=""       # minimum frequency
+n_mels=80     # number of mel basis
+n_fft=1024    # number of fft points
+n_shift=256   # number of shift points
+win_length="" # window length
```

Comment:
How about `n_fft=2048`, `n_shift=300` and `win_length=1200` by following the Tacotron2 paper? If the sampling frequency is the same as experiments in Tacotron 2 (i.e. 24kHz), it would be nice to follow the acoustic feature extraction parameters in the paper unless we find other parameters that work better. Also increasing the n_shift reduces the decoder target lengths so I think it would make training seq2seq models slightly easier.

> https://arxiv.org/pdf/1712.05884.pdf
> As in Tacotron, mel spectrograms are computed through a short-time Fourier transform (STFT) using a 50 ms frame size, 12.5 ms frame hop, and a Hann window function.

Reply:
Thank you for your suggestion. I will try that.

Reply:
As you said, training speed became faster in the condition. The synthesized speech generated by the model trained on the condition was clear. This may be because `n_fft=2048`. However, the prosody was slightly worse and the power was quite low (I'm not sure why). I will continue to find good settings.

Reply:
Sorry to comment on the merged PR, but let me confirm the conclusion. Do you think we should stick to use n_fft=1024 and so on? If you think it's difficult to say one is clearly better than the other or vise versa, we may consider using ASR-based objective evaluation (#1190) to investigate differences in a measurable way.

Reply:
Good idea. Let me try to objectively evaluate synthesized speech when the PR is merged.